src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -150,22 +147,12 @@ def __init__(self, *args, **kwargs):
 
     def setUp(self):
         """Set up each test case."""
-        # get paths from the build_vars generated by build
-        try:
-            with open('../../.build_vars.json') as build_vars:
-                build_paths = json.load(build_vars)
-            self.basepath = os.path.normpath(os.path.join(build_paths['PREFIX'],
-                                                          '..') + os.path.sep)
-            self.prefix = build_paths['PREFIX']
-            try:
-                self.ofi_prefix = build_paths['OFI_PREFIX']
-            except KeyError:
-                self.ofi_prefix = os.sep + "usr"
-        except FileNotFoundError:
-            self.prefix = "/usr"
-            self.basepath = "/"
-            self.ofi_prefix = os.sep + "usr"
-            self.log.info("No build vars file, assuming RPM install")
+        # Assume PREFIX relative to this installed file
+        self.prefix = os.path.normpath(
+            os.path.join(
+                os.path.dirname(os.path.realpath(__file__)),
+                "..", "..", "..", "..", "..", "..", ".."))
+        self.log.info("Assuming daos install prefix = %s", self.prefix)
```

Comment:
I wonder if we should just use something like this to get the install path
```
command -v daos
```
it would more easily support running launch.py from a source build, but against an RPM install because the user would just need to export PATH, etc.